journey-map/public/render.mjs

```diff
@@ -35,20 +35,25 @@ export const getOptions = (journeyMaps) => {
 };
 
 // Expand out parent states
-const expandParents = (journeyMap) => {
+// Will also search 'otherStates' (e.g. for nested journeys)
```

Comment:
This allows us to see the failure states etc. inherited by `CRI_STATE` depending on the context.

In future we probably want to avoid this pattern - it's pretty confusing that the nested journey can inherit things from the journey it's in, but it works like that for now.